frontend/www/js/omegaup/arena/course.ts

```diff
@@ -53,6 +64,55 @@ OmegaUp.on('ready', () => {
               problems: this.problems,
             });
           },
+          'reset-hash': (request: { selectedTab: string; alias: string }) => {
```

Comment:
nit:

```suggestion
          'show-problem': (request: { selectedTab: string; alias: string }) => {
```

reset suena a que lo va a regresar a su estado original (sin hash? el primer hash que usó el usuario?). y además, es mejor si en vez de decir qué es lo que está cambiando en el componente en vez de lo que el componente quiere que se haga (declarativo vs. imperativo).

también, como ya se está mandando el selectedTab via `update:activeTab` abajo, se puede únicamente mandar el alias?

Reply:
Como no se están usando los tabs de `ProblemDetails`, estoy quitando todo este evento. CC. @pabo99

Reply:
Parece que este evento lo que controla el hash de los tabs del curso/concurso (que son muy similares)